docs/README.md

```diff
@@ -1171,6 +1171,7 @@ Use Docker to build and serve the content, but remember to rebuild the Docker im
 - Images should be placed under the root `img` folder and referred to using HTML `<img>` tags
 - H3 header (`###`) should be the default header within a page
 - H2 header (`##`) will appear in the navigation as the page's table of contents
+- To add a new section, add a new file to `docs` such as `x.x-new-section.md`, and then reference it in `docs/_sidebar.md`.  Please note: this will also generate new front matter in `docs/README.md`.
```

Comment:
Yes and no. Adding a new section by itself does not mean new front matter will be generated. That is only the case if the new section contains front matter. The front matter also feels like an aside and probably worth writing more about why it is even there, what purpose it serves, when it is needed and needs to be updated etc.

Reply:
Okay, I went ahead and removed the note on the front matter.  The main reason I opened this PR was to set future contributors up with some basic instructions for adding a new Bootcamp section, so the front matter notes should probably be addressed separately.